rest-api/tools/deploy_app.sh

```diff
@@ -42,7 +42,7 @@ if [[ $(git status --porcelain) ]]; then
 fi
 
 UPDATE_TRACKER=tools/update_release_tracker.py
-if [ "${PROJECT}" == "all-of-us-rdr-prod" ]
+if [ "${PROJECT}" == "all-of-us-rdr-prod" ] || [ "${PROJECT}" == "all-of-us-rdr-staging" ]
```

Comment:
This should change to use config_staging.json... we don't want to use prod config, just app_prod.yaml.

Reply:
done